easybuild/easyconfigs/c/CRIU/CRIU-3.13-GCC-8.3.0-Python-3.7.4.eb

```diff
@@ -0,0 +1,71 @@
+# #
+# This is a contribution from HPCNow! (http://hpcnow.com)
+# Copyright::   HPCNow!
+# Authors::     Jordi Blasco <jordi.blasco@hpcnow.com>
+# License::     GPL-v3.0
+# #
+
+easyblock = 'ConfigureMake'
+
+name = 'CRIU'
+version = '3.13'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/checkpoint-restore/criu'
+description = """Checkpoint/Restore In Userspace (CRIU) is a Linux software which can freeze a running container
+(or an individual application) and checkpoint its state to disk. The data saved can be used to restore the 
+application and run it exactly as it was during the time of the freeze. Using this functionality, application 
+or container live migration, snapshots, remote debugging, and many other things are now possible."""
+
+toolchain = {'name': 'GCC', 'version': '8.3.0'}
+toolchainopts = {'pic': True}
+
+builddependencies = [
+    ('Autotools', '20180311'),
+    ('protobuf', '3.10.0'),
+    ('protobuf-c', '1.3.3'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('Python', '3.7.4'),
+    ('future', '0.16.0', versionsuffix),
```

Comment:
`future` is already included in the `Python` 3.7.4 easyconfig, so this extra dependency is probably overkill?